src/effects/VST3/VST3ParametersWindow.cpp

```diff
@@ -0,0 +1,187 @@
+/**********************************************************************
```

Comment:
The commit that introduces this file makes a dependency cycle between it and VST3Utils.h

The easy solution, I think, is to move `BuildPlainUI` into the new file.  Then the new file depends one-way on VST3Utils.